### DISCUSSION TITLE
How to run multiple runs in a single project

### DISCUSSION THREAD

**Discussion by Maduranga** [sorry-cypress/sorry-cypress#511] [Q&A] [answered]
Hello,

I am running my Cypress tests in a Jenkins server and then sending the results to `sorry-cypress` dashboard.
This works perfectly the first time. However, when I run the same test again, it does not run the tests. This is because there is already a run in the project. If I delete that run from dashboard, then I can run the tests again without problem.

When I run the very first run (of a project), this is the initial output:
```
================================================================================

  (Run Starting)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Cypress:        9.1.1                                                                          │
  │ Browser:        Chrome 83 (headless)                                                           │
  │ Node Version:   v16.13.1 (/usr/local/bin/node)                                                 │
  │ Specs:          25 found (widget_test_for_clients/clien...)                                                    │
  │ Searched:       cypress/tests/widget_test_for_clients/*.js                                     │
  │ Params:         Tag: false, Group: false, Parallel: false                                      │
  │ Run URL:        http://cy-xxxxxxx.us-west-2.elb.amazonaws.com/run/2445645745244234 │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
```

The problem I believe is the `Run URL`. The token at the end is the same for each run, which refers to the run in the dashabord, and if there is a run with that token, the test is not run. 
How can I execute each test so that the `Run URL` is different?
I have tried to run tests with different `--record --key` but it didn't help.

I know this is possible to do, but I cannot figure out how to.

**Comment by agoldis** [comment, marked as answer]
I remember there was an issues / discussion about this. 
If you generate a different ciBuildId when rerunning, you'll get a new run. 

Not sure if Jenkins provides an env variable the uniquely identifies reruns

**Comment by Maduranga** [comment]
Thank you.
What worked was passing something like
-`-record --key 12345 --ci-build-id abcd --group test-1`
